rust/api-server/migration/src/m20241208_112015_inventory_changelog.rs

```diff
@@ -95,7 +95,7 @@ impl MigrationTrait for Migration {
         let stmt = Statement::from_string(
             manager.get_database_backend(),
             format!(
-                "SELECT create_hypertable('{}', by_range('{}'));",
```

Comment:
Did they change something in timescale db or what did fail for you? Or did I do it wrong from the start?